glusterfs.spec.in

```diff
@@ -980,8 +985,8 @@ touch %{buildroot}%{_sharedstatedir}/glusterd/nfs/run/nfs.pid
 find ./tests ./run-tests.sh -type f | cpio -pd %{buildroot}%{_prefix}/share/glusterfs
 
 ## Install bash completion for cli
-install -p -m 0744 -D extras/command-completion/gluster.bash \
-    %{buildroot}%{_sysconfdir}/bash_completion.d/gluster
+install -p -m 0644 -D extras/command-completion/gluster.bash \
```

Comment:
you might want to do this in the `extras/command-completion/Makefile.in` (but only if the CLI is enabled)?

Reply:
The CLI seems to always be installed and the completion script is installed as data (with `0644`) from the `extras/command-completion/Makefile`.

With the latest commit and the completion script installed by default, this install call is no longer needed in the spec file.